Fibonacci_Number/Fibonacci_NumberDP.c

```diff
@@ -0,0 +1,51 @@
+#include <stdio.h>
+int dp[100005] = {0};
+
+int fibDP(int n)
+{
+    if(n == 0 || n == 1)
+    {
+        return n;
+    }
+    /* This is used to print the values or series of given fibonacci number */
+    /*if(dp[n] != 0)
+    {
+        return dp[n];
+    }*/
+    
+    /* This recursion is to hold the values till cand1 = 1(from 5-4-3-2-1), cand2 = 1(5-3-1) */
+    int cand1 = fibDP(n - 1);
+    int cand2 = fibDP(n - 2);
+    /* Later, it will go back to its previous values(2-3), cand1, cand2 values will be updated.
+    /* after, getting values (1,1), those values get added and updated at 5th index. this will repeat in each iteration*/
+    /* you can check(print) manually the values of cand1, cand2, n and dp[n] at multiple instances to get a proper idea of structure of the program */
```

Comment:
make this as a multi line comment
one line should contain only 80 characters 
please changes this in all the other files too

Reply:
Yes, changed it to multi-line comment. please chack it.